lib/scorecard.rb

```diff
@@ -0,0 +1,54 @@
+class Scorecard
```

Comment:
Should a scorecard class be responsible for adding a score, storing frame numbers, rolling?